main.go

```diff
@@ -41,6 +41,7 @@ var (
 	encode  = flag.Bool("io", true, "create Encode and Decode methods")
 	marshal = flag.Bool("marshal", true, "create Marshal and Unmarshal methods")
 	tests   = flag.Bool("tests", true, "create tests and benchmarks")
+	exports = flag.Bool("exports", true, "process only exported types")
```

Comment:
Personally, I'd rather this be the other way. Otherwise usage is a little awkward. For example:

```
msgp -exports=false ...
```

is far less useable/readable as opposed to if you defined the flag to be something along the lines:

```go
var allTypes = flag.Bool("all-types", false, "process all types")
```

which means usage would be

```
msgp -all-types ...
```

which is far more readable IMHO, but that's just my two cents.